pandas/conftest.py

```diff
@@ -881,3 +881,75 @@ def index_or_series(request):
     See GH#29725
     """
     return request.param
+
+
+_int_index = tm.makeIntIndex(10, name="a")
```

Comment:
Are these fixtures used outside of `base`? If not, then should just create a conftest in that directory to house

Reply:
Yes, they will be used as well in:
- `pandas/tests/indexes/datetimes`
- `pandas/tests/indexes/period`
- `pandas/tests/indexes/timedeltas`